packages/common/src/visualizations/CartesianChartDataModel.ts

```diff
@@ -119,7 +119,25 @@ export class CartesianChartDataModel {
 
     // Get the formatter for the value label,
     // which has more complex inputs
-    static getValueFormatter(format: Format | undefined) {
+    static getValueFormatter(
```

Comment:
these changes are for sql runner charts

Reply:
Would it make sense to call this something like `usePercents` rather than `isStack100`? If ever we wanted to use this for something similar.